bin/bridge-client/src/modes/operator/bootstrap.rs

```diff
@@ -136,23 +146,29 @@ pub(crate) async fn bootstrap(args: Cli) -> anyhow::Result<()> {
         msg_polling_interval,
     };
 
+    let max_retry_count = args.max_duty_retries.unwrap_or(DEFAULT_DUTY_RETRY_COUNT);
     let task_manager = TaskManager {
         exec_handler: Arc::new(exec_handler),
         broadcaster: l1_rpc_client,
         bridge_duty_db_ops,
         bridge_duty_idx_db_ops,
+        max_retry_count,
```

Comment:
See prev comment: https://github.com/alpenlabs/strata/pull/394#discussion_r1793788011

reply: 

> It's just this one right now so introducing a struct is a bit of an overkill. But will definitely do that once this starts to become untenable.

It's not overkill, this *is* the time to do it since we're newly adding this kind of configuration.  When we need to introduce additional fields in the future we *won't* have to refactor the arguments in every place it's used to replace the `max_retry_count` with the more general `config`.

Reply:
Done in https://github.com/alpenlabs/strata/pull/398/commits/66b251ecf8d30ae46d57b1171ca2a6bc8c94fa5a.